### ISSUE TITLE
Question about the gradient of positions 

### ISSUE THREAD

**Issue by smiles724** [TUM-DAML/gemnet_pytorch#6]
Hi, great work, and thanks for sharing the code. I have a small question regarding the gradient of inputs["R"]. 
As far as I know, inputs["R"] represents the positions of atoms. Why do you make its requires_grad True?

```
        if not self.direct_forces:
            inputs["R"].requires_grad = True
```

**Comment by gasteigerjo** [comment]
If we don't predict forces directly (`self.direct_forces=False`), we calculate them via F = -dE/dx. This thus requires exactly the positions' gradients.
